WalletWasabi.Fluent/UiConfig.cs

```diff
@@ -22,6 +19,7 @@ public class UiConfig : ConfigBase
 	private string _windowState = "Normal";
 	private bool _runOnSystemStartup;
 	private bool _oobe;
+	private Version _lastVersionHighlightsDisplayed = new (2, 3, 1);
```

Comment:
maybe better to have the client version constant?
to not have to edit this, and it might be forgotten to edit etc